draft-ietf-mops-streaming-opcons.md

```diff
@@ -187,6 +193,47 @@ under peak load?
 
 Provisioning issues for caching systems?
 
+## Personalization and Advertizing
+
+Some of this fits better under adaptive bitrate, but most of it seems
```

Comment:
It's probably a good idea to mark this kind of paragraph as TBD and/or [EDITOR: remove this paragraph], to make sure it doesn't accidentally end up in the final text.

Also: I'm thinking maybe the key personalization content fits better as a new top-level section after Adaptive Bitrate, rather than  inside the "Bandwidth Provisioning" section?